packages/api/src/graphql/comment/comment.private-mutation.ts

```diff
@@ -39,10 +50,16 @@ export const updateComment = async (
   return commentClient.update({
     where: {id: commentId},
     data: {
-      revisions: text
+      userID,
+      guestUsername,
+      guestUserImageID,
+      source,
+      revisions: revision
         ? {
             create: {
-              text
+              text: (revision.text as unknown) as string,
```

Comment:
I know I had type this as string before which is wrong.
Does Prisma.JsonValue work? :(